frontend/src/metabase/query_builder/components/view/sidebars/QuestionInfoSidebar/tests/common.unit.spec.tsx

```diff
@@ -0,0 +1,83 @@
+import userEvent from "@testing-library/user-event";
+import {
+  createMockCard,
+  createMockModerationReview,
+} from "metabase-types/api/mocks";
+import { screen } from "__support__/ui";
+import { setup } from "./setup";
+
+describe("QuestionInfoSidebar", () => {
+  describe("description", () => {
+    it.each([
+      createMockCard({ name: "Question", description: "abc", dataset: false }),
+      createMockCard({ name: "Model", description: "abc", dataset: true }),
+    ])("should display description of a $card.name", async card => {
```

Comment:
the test names aren't displayed correctly.

```suggestion
    ])("should display description of a $name", async card => {
```
#### Before
![image](https://github.com/metabase/metabase/assets/1937582/63a17e8f-c098-42e4-b6a2-bebe83515911)

#### After
![image](https://github.com/metabase/metabase/assets/1937582/d2e42875-dec3-49c3-9f09-2d14fa6608df)

Reply:
Fixed